posts/2021-08-31-open-liberty-starter.adoc

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Introducing the Open Liberty Starter!"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/steven1046
+author_github: https://github.com/steven1046
+seo-title: An easy way to generate the necessary files to start with Open Liberty - OpenLiberty.io
+seo-description: An easy way to generate the necessary files to start with Open Liberty.
+blog_description: An easy way to generate the necessary files to start with Open Liberty.
+open-graph-image: https://openliberty.io/img/blog/starter_application.png
+---
+= Introducing the Open Liberty Starter!
+Steven Zvonek <https://github.com/steven1046>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+== Open Liberty Starter
+
+There is now another easy way to get started with Open Liberty. Visit the link:{url-prefix}/downloads/[Getting Started] page and select a few options to generate a starter application that runs on Open Liberty. You can specify your application and project name, choose a build tool from either Maven or Gradle, and pick which version of Java SE, Jakarta EE, and MicroProfile your application will use. Then, just click `Generate Project` and you are ready for lift-off! 
+
+=== What is the starter?
+
+The Open Liberty starter gives you a simple, quick way to get the necessary files to start building an application on Open Liberty. There is no need to search how to find out what to add to your maven or gradle build files. A simple RestApplication.java gets generated for you to start creating a project. A server.xml configuration file gets generated for you with the necessary features for the MicroProfile and Jakarta EE versions that you pick. A Dockerfile is generated for you to easily create a new Docker image of your project on top of Open Liberty's Docker image.
```

Comment:
A simple RestApplication.java gets generated for you to start creating a project. A server.xml configuration file gets generated for you with the necessary features for the MicroProfile and Jakarta EE versions that you pick.
---->
A simple `RestApplication.java` file iss generated for you to start creating a project. A `server.xml`  configuration file is also generated  with the necessary features for the MicroProfile and Jakarta EE versions that you selected during set up.